modules/ROOT/examples/live-demos/full-featured/index.js

```diff
@@ -208,6 +208,8 @@ tinymce.ScriptLoader.loadScripts(['https://cdn.jsdelivr.net/npm/faker@5/dist/fak
     mentions_menu_hover: mentions_menu_hover,
     mentions_menu_complete: mentions_menu_complete,
     mentions_select: mentions_select,
-    mentions_item_type: 'profile'
+    mentions_item_type: 'profile',
+    autocorrect_autocorrect: true,
+    autocorrect_capitalize: true,
```

Comment:
```suggestion
    mentions_item_type: 'profile'
```
This shouldn't have been added as these are the defaults. It should only be added if it's changing something from the defaults.